src/shared/cli/Runner.php

```diff
@@ -86,7 +86,7 @@ public function run(): int {
     /**
      * @throws ErrorException
      */
-    public function errorHandler(int $code, string $message, string $file, int $line, array $context): void {
+    public function errorHandler(int $code, string $message, string $file, int $line, array $context): bool {
```

Comment:
This is technically incorrect.

Reply:
I know that from an implementation perspective this is incorrect. But according to the php docs a errorhandle method must return a `bool` https://www.php.net/manual/en/function.set-error-handler.php

> If the function returns FALSE then the normal error handler continues.

Reply:
Point taken.